bin/docserv

```diff
@@ -42,7 +42,6 @@ class RepoLock:
         self.thread_id = thread_id
 
     def acquire(self, blocking=True):
-        logger.debug("Thread %i: Acquiring lock %s." % (self.thread_id, self.resource_name))
```

Comment:
Are you removing that because you think it is useless or because you have a similar log message elsewhere?

Reply:
Yeah, logging which locks are acquired and released is a little too much. It was interesting to see if there are any problems with the locks. But now I'm confident enough that it works as it should.